test/unit/styling-cross-scope-apply.html

```diff
@@ -736,6 +776,17 @@
       assert.equal(parent2Text, parent3Text, 'x-produce-mixin-3 should not have invalidated x-consume-mixin');
     }
   });
+
+  test('mixins can be depended on before creation', function() {
+    var e = document.createElement('x-apply-depend-before-create');
+    document.body.appendChild(e);
+    CustomElements.takeRecords();
+    var e2 = document.createElement('x-create-after-apply');
+    document.body.appendChild(e2);
+    CustomElements.takeRecords();
+    assertComputed(e2.$.child, '2px');
+    assertComputed(e2.$.child, 'rgb(0, 0, 255)', 'color');
```

Comment:
Let's make a separate test for this assertion since it's testing that variable values that are mixins elsewhere function correctly.